devops/containers/ubuntu2004_intel_drivers.Dockerfile

```diff
@@ -0,0 +1,58 @@
+ARG base_tag=latest
+ARG base_image=ghcr.io/intel/llvm/ubuntu2004_base
+
+FROM $base_image:$base_tag
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+ARG compute_runtime_tag=latest
+ARG igc_tag=latest
+ARG tbb_tag=latest
+ARG fpgaemu_tag=latest
+ARG cpu_tag=latest
+
+RUN apt update && apt install -yqq wget
+
+COPY scripts/get_release.py /
+
+# Install IGC
+RUN python3 /get_release.py intel/intel-graphics-compiler $igc_tag \
+  | grep ".*deb" \
+  | wget -qi - && \
+  dpkg -i *.deb && rm *.deb
+
+# Install NEO
+RUN python3 /get_release.py intel/compute-runtime $compute_runtime_tag \
```

Comment:
is ocloc installed as part of this installation?

Reply:
Yes, it's part of `intel-opencl-icd_smth.deb`